scss/components/_buttons.scss

```diff
@@ -0,0 +1,144 @@
+//header_button
+
+header {
```

Comment:
wtf??? why you have header section here...
przeciez tworzysz komponenty -> reuzywalne elementy czemu zamykasz  te style na tylko jedna sekcje ....